app/views/evaluations/submissions.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= render 'shared/back_link', path: evaluations_path %>
+
+<h1><%= challenge_phase_title(@phase.challenge, @phase) %></h1>
+<p class="text-normal">View challenge submissions and manage evaluation progress.</p>
```

Comment:
figma now reads:
```suggestion
<p class="text-normal">View challenges with submissions assigned to me to evaluate.</p>
```
(mobile design is slightly different but let's just use this for now)